webrender/src/renderer.rs

```diff
@@ -873,8 +915,27 @@ impl<B: hal::Backend> GpuCacheTexture<B> {
         }
     }
 
-    fn update(&mut self, _device: &mut Device<B>, updates: &GpuCacheUpdateList) {
+    fn update(&mut self, device: &mut Device<B>, updates: &GpuCacheUpdateList) {
         match self.bus {
+            #[cfg(not(feature = "gleam"))]
+            GpuCacheBus::PMbuffer { ref mut mapped_ranges } => {
+                let mut writer = device.create_writer();
+                for update in &updates.updates {
+                    match *update {
+                        GpuCacheUpdate::Copy {
+                            block_index,
+                            block_count,
+                            address,
+                        } => {
+                            let address = address.v as usize * MAX_VERTEX_TEXTURE_WIDTH + address.u as usize;
+                            mapped_ranges.push(address as u64 * 4 .. (address + block_count) as u64 * 4);
```

Comment:
why are we multiplying by `4`? The block size here is 4 floats, which is 16 bytes

Reply:
That's right, got confused by the `[f32; 4]`, fixed it here: https://github.com/szeged/webrender/pull/297/commits/61242837ca2384876662a54d0bc03f7d9e4727c5#diff-f5062b694b9fe53fc1757ed483d577d9R931 . Also adding a note to myself to use `GpuBlockData` for the cache buffer related things instead of `[f32; 4]`.